pkg/controller/replication_controller.go

```diff
@@ -204,6 +204,10 @@ func (rm *ReplicationManager) getPodControllers(pod *api.Pod) *api.ReplicationCo
 // When a pod is created, enqueue the controller that manages it and update it's expectations.
 func (rm *ReplicationManager) addPod(obj interface{}) {
 	pod := obj.(*api.Pod)
+	if pod.DeletionTimestamp != nil {
```

Comment:
Why aren't we also filtering out terminated pods here? Could we factor out the test from filterActivePods and use that in both places?